helm-v3/006-repositories.md

```diff
@@ -2,10 +2,22 @@
 
 The following changes will be made to the repository subsystem:
 
+- A new `helm login` command will be added for authenticating against a repository
+- A new `helm push` command will be added for pushing charts to a repository
 - The `helm serve` command will be removed
-- A new `helm push` verb will be added for pushing charts to a repository
+- [ChartMuseum](https://github.com/helm/chartmuseum) will be the Helm subproject
+  dedicated to hosting your own repository
```

Comment:
A couple thoughts:

* This document is about Helm (the CLI) v3. Does the scope of other sub-projects belong in here?
* Helm (the cli) provides a means to generate a static repo. External tools, such as artifactory and S3, can be repos. ChartMuseum is the Helm projects server for a repository, right? How do we communicate that clearly?

Would it be better to say that `helm login` and `helm push` will, by default, use the specification to communicate with ChartMuseum, the repo server provided by the Helm project and that the spec for this and extension mechanism for those commands will be documented?